lib/html_parser.dart

```diff
@@ -288,7 +304,28 @@ class HtmlParser extends StatelessWidget {
       tree: tree,
       style: context.style.copyOnlyInherited(tree.style),
     );
-
+    List<int> indices = [];
```

Comment:
The name `indices` is very generic. I think these are the indices where we want to insert line endings, right? So maybe call it `lineEndingIndices`?

Reply:
Fixed.